library/VTKExtensions/Rendering/vtkF3DRenderer.cxx

```diff
@@ -751,12 +757,15 @@ void vtkF3DRenderer::ConfigureHDRITexture()
     }
     else
     {
-      // Reading the HDRI texture is actually not needed
-      // create a dummy one instead
-      // TODO add support for not providing a texture and still using IBL in VTK
-      // https://github.com/f3d-app/f3d/issues/935
+#if VTK_VERSION_NUMBER >= VTK_VERSION_CHECK(9, 3, 20230830) // TODO date
```

Comment:
> // TODO date

It's not merged in VTK yet?

Reply:
No, I'm making myself ready to merge everything together.